### ISSUE TITLE
🐛 Bug Report: Email setup appears to be broken in 0.5.1

### ISSUE THREAD

**Issue by jl-678** [stonith404/pingvin-share#72]
### 👟 Reproduction steps

I broke my installation and so decided to reinstall.  (This was due to a permissions issue.) Prior to reinstalling, I used 0.5.0 and everything worked as expected.  

As part of the rebuild, I removed all persistent volumes and reinstalled via docker compose.  Everything went smoothly. I then went through the initial user setup process without an issue.

Post install, I choose "administration" and then "configuration" and go and setup SMTP credentials using known good credentials.

I input my SMTP password after setting up everything else and it will not stay.  As soon as I click save and refresh the page, the credentials are gone.  Checking out the SQLite database, I see that the credentials are not stored.

### 👍 Expected behavior

The SMTP password should be accepted and just work.

### 👎 Actual Behavior

The password disappears and as a result, emails fail.  Here is what the logs show:

```
[Nest] 203  - 01/04/2023, 6:47:48 PM   ERROR [ExceptionsHandler] Missing credentials for "PLAIN"
Error: Missing credentials for "PLAIN"
    at SMTPConnection._formatError (/opt/app/backend/node_modules/nodemailer/lib/smtp-connection/index.js:790:19)
    at SMTPConnection.login (/opt/app/backend/node_modules/nodemailer/lib/smtp-connection/index.js:444:38)
    at /opt/app/backend/node_modules/nodemailer/lib/smtp-transport/index.js:272:32
    at SMTPConnection.<anonymous> (/opt/app/backend/node_modules/nodemailer/lib/smtp-connection/index.js:213:17)
    at Object.onceWrapper (node:events:627:28)
    at SMTPConnection.emit (node:events:513:28)
    at SMTPConnection._actionEHLO (/opt/app/backend/node_modules/nodemailer/lib/smtp-connection/index.js:1325:14)
    at SMTPConnection._processResponse (/opt/app/backend/node_modules/nodemailer/lib/smtp-connection/index.js:953:20)
    at SMTPConnection._onData (/opt/app/backend/node_modules/nodemailer/lib/smtp-connection/index.js:755:14)
    at SMTPConnection._onSocketData (/opt/app/backend/node_modules/nodemailer/lib/smtp-connection/index.js:193:44)
```

I tried adding my password by manually updating SQLite and then get a weird error with a failure trying to connect to 127.0.0.1 which is strange because I have an SMTP URL set.

I saw that you updated the email config in 0.5.1 and so am wondering if this caused that issue.

### 🌐 Browser

Chrome

**Comment by jl-678** [comment]
Now it seems to work.  No idea why. I will close this.

**Comment by stonith404** [comment]
That's strange, did it work after you restarted the container?

**Comment by jl-678** [comment]
I am not entirely sure what I did. It must have been some combination of restarting and re-pulling the container.  It was strange though because I thought that all persistent data was on the local volume and I definitely wiped that. (I actually moved it and then created a new, empty one.)

I was always careful to stop the container when monkeying with the storage volume.  Who knows that.  It was odd.  Anyway, sorry to bother you.
